tests/components/unifi/test_sensor.py

```diff
@@ -131,12 +135,10 @@ async def test_sensors(hass):
 async def test_remove_sensors(hass):
     """Test the remove_items function with some clients."""
     controller = await setup_unifi_integration(
-        hass,
-        options={unifi.const.CONF_ALLOW_BANDWIDTH_SENSORS: True},
-        clients_response=CLIENTS,
+        hass, options={CONF_ALLOW_BANDWIDTH_SENSORS: True}, clients_response=CLIENTS,
     )
-    assert len(hass.states.async_entity_ids("sensor")) == 4
-    assert len(hass.states.async_entity_ids("device_tracker")) == 2
+    assert len(hass.states.async_entity_ids(SENSOR_DOMAIN)) == 4
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)